docs/website/content/docs/v0.1/Configuration/metadata.md

```diff
@@ -4,3 +4,29 @@ weight: 4
 ---
 
 # Metadata
+
+The Metadata server manages the Machine metadata.  Critically, this includes the
+production of the configuration of Talos to be used during the automated installation.
+
+## Talos Machine Configuration
+
+The configuration of each machine is constructed from a number of sources:
+
+  - The `Cluster` of which the `Machine` is a member.
+  - The `ServerClass` which was used to select the `Server` into the `Cluster`.
+  - Any `Server`-specific patches.
+
+The base template is constructed from the Cluster.  Then, any configuration
+patches are applied from the `ServerClass` and `Server`.
+
+Only configuration patches are allowed in the `ServerClass` and `Server`
+resources.  These patches take the form of an [RFC 6902](https://tools.ietf.org/html/rfc6902) JSON (or YAML) patch.
```

Comment:
May want to point [here](https://www.sidero.dev/docs/v0.1/configuration/servers) with a note saying you can see an example.